src/core/thread/osthread.d

```diff
@@ -3332,15 +3332,17 @@ extern(C) void thread_processGCMarks( scope IsMarkedDg isMarked ) nothrow
 
 extern (C) @nogc nothrow
 {
-    version (CRuntime_Glibc) int pthread_getattr_np(pthread_t thread, pthread_attr_t* attr);
-    version (FreeBSD) int pthread_attr_get_np(pthread_t thread, pthread_attr_t* attr);
-    version (NetBSD) int pthread_attr_get_np(pthread_t thread, pthread_attr_t* attr);
-    version (OpenBSD) int pthread_stackseg_np(pthread_t thread, stack_t* sinfo);
-    version (DragonFlyBSD) int pthread_attr_get_np(pthread_t thread, pthread_attr_t* attr);
-    version (Solaris) int thr_stksegment(stack_t* stk);
-    version (CRuntime_Bionic) int pthread_getattr_np(pthread_t thid, pthread_attr_t* attr);
-    version (CRuntime_Musl) int pthread_getattr_np(pthread_t, pthread_attr_t*);
-    version (CRuntime_UClibc) int pthread_getattr_np(pthread_t thread, pthread_attr_t* attr);
+    version (CRuntime_Glibc)  version = PThread_Getattr_NP;
+    version (CRuntime_Bionic) version = PThread_Getattr_NP;
+    version (CRuntime_Musl)   version = PThread_Getattr_NP;
+    version (CRuntime_UClibc) version = PThread_Getattr_NP;
+
+    version (FreeBSD)         version = PThread_Attr_Get_NP;
+    version (NetBSD)          version = PThread_Attr_Get_NP;
+    version (DragonFlyBSD)    version = PThread_Attr_Get_NP;
```

Comment:
Should we have a `else static assert(0, "Unsupported platform")` ?

Reply:
That is still handled in https://github.com/dlang/druntime/pull/2924/files#diff-f58c556db26f85763b2658851d550455L3487

Reply:
Not here, as that would trigger on OSX and windows ;)